ikenna-portfolio/portfolio/src/test/java/com/google/sps/DatabaseFunctionsTest.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.sps.data.Comment;
+import com.google.sps.database.*;
+import java.io.*;
+import javax.servlet.http.*;
+import org.junit.Test;
+import org.mockito.Mockito;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class DatabaseFunctionsTest extends Mockito {
+  /*
```

Comment:
Please add these tests back after you fix the unit test issue.

Reply:
done